plugins/builds/index.js

```diff
@@ -11,10 +11,98 @@ const listSecretsRoute = require('./listSecrets');
 const tokenRoute = require('./token');
 const metricsRoute = require('./metrics');
 const workflowParser = require('screwdriver-workflow-parser');
+const deepmerge = require('deepmerge');
+const schema = require('screwdriver-data-schema');
+const { EXTERNAL_TRIGGER_AND } = schema.config.regex;
 
 /**
- * Create the build. If config.start is false or not passed in then do not start the job
- * @method createBuild
+ * Create event for downstream pipeline that need to be rebuilt
+ * @method createEvent
```

Comment:
this function is the same as previous `triggerEvent`
factored it out since we need to call it somewhere else.